### PR TITLE
Working 2016

### PR DESCRIPTION
**Pull request by bookdude13**

### CODE REVIEW

**Comment by computergeek125** [comment]
Seen, but will deal with after homework

**Comment by bookdude13** [comment]
Fair, was just going through and thought I'd assign.

**Comment by computergeek125** [comment]
Whoops.  Well this merge is going to be interesting.  (will work on it today)

File: Session.vim [hunk 1]
```diff
@@ -0,0 +1,50 @@
+let SessionLoad = 1
```
Comment (computergeek125) [inline Session.vim]: Session.vim?
was this a `git commit -a` per chance?

**Review by computergeek125 — APPROVED** [review]
Looks good.

**Comment by bookdude13** [comment]
Removed that Session.vim, and also removed the Music/ folder, since it does not appear to be used anywhere.

**Pull request merged**